<launch>
    <!-- Set the ROS package and node name -->
    <node pkg="calibration_odom_based" type="calibration_odom_based" name="calibration_odom_based" output="screen">
        <!-- Set any required ROS parameters -->
        <!-- Odom1 is reference frame for calibration -->
        <param name="odom1_topic" value="/odom1_topic" />
        <param name="odom2_topic" value="/odom2_topic" />

        <!-- Set the type of the odometry messages -->
        <!-- odom type can be 
            ODOM_TYPE_NAV_MSGS_ODOM: nav_msgs::Odometry |
            ODOM_TYPE_GEO_MSGS_TWIST: geometry_msgs::TwistStamped |
            ODOM_TYPE_GEO_MSGS_POSESTAMPED: geometry_msgs::PoseStamped
        -->
        <param name="odom1_type" value="ODOM_TYPE_NAV_MSGS_ODOM" /> 
        <param name="odom2_type" value="ODOM_TYPE_NAV_MSGS_ODOM" />

        <!-- Set the path to the configuration file -->
        <param name="ini_path" value="$(find calibration_odom_based)/config/odom_based_calibration.ini" />
    </node>
</launch>